### PR TITLE
Feature/update workflows to react to tags

### PR DESCRIPTION
**Pull request by saschadoemer**

### CODE REVIEW

File: agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs [hunk 1]
```diff
@@ -245,8 +247,7 @@ public EncodedMessage Encode(SendChunkedMessageParameters sendMessageParameters)
                 TechnicalMessageType = sendMessageParameters.TechnicalMessageType,
                 Mode = Mode,
                 Recipients = sendMessageParameters.Recipients,
-                ChunkInfo = sendMessageParameters.ChunkInfo,
-                Metadata = sendMessageParameters.Metadata,
+                ChunkInfo = sendMessageParameters.ChunkInfo
```
Comment (Frank-Wiebeler) [inline agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs:250]: Why was the MetaData-Field removed; Following the  documentation (see https://docs.my-agrirouter.com/agrirouter-interface-documentation/latest/integration/build-message.html ), every chunk shall include the information

**Review by Frank-Wiebeler — CHANGES_REQUESTED** [review]
Thank you for the input, found a small issue to be fixed before merge. :)

File: agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs (hunk 1 above)
Comment (oliverrahner) [reply agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs:250]: It has only been added yesterday, as #101 had been merged :) Probably this branch isn't totally up-to-date?

**Review by Frank-Wiebeler — APPROVED** [review]
Awesome, thank you for the input ☜(⌒▽⌒)☞

**Pull request merged**